modules/openy_gc_auth/modules/openy_gc_auth_example/config/install/openy_gc_auth.provider.dummy.yml

```diff
@@ -0,0 +1,5 @@
+id: dummy
+user: dummy@test.com
+token: 7Ln3ljjlh8TA9N96cCNiLnxvP1m79MdL
+password: password
+redirect_url: 'http://rose.openy.docksal/dummy/provider/redirect'
```

Comment:
is this correct?

Reply:
yes, this is just an example of configuration, redirect_url from this settings not used